routing/router.go

```diff
@@ -1252,6 +1228,107 @@ func (r *ChannelRouter) networkHandler() {
 	}
 }
 
+func (r *ChannelRouter) getMissingBlocks(currentHeight uint32,
+	chainUpdate *chainview.FilteredBlock) {
+
+	outdatedHash, err := r.cfg.Chain.GetBlockHash(int64(currentHeight))
+	if err != nil {
+		log.Error("unable to retrieve outdated block hash")
+	}
+
+	outdatedBlock := &chainntnfs.BlockEpoch{
+		Height: int32(currentHeight),
+		Hash:   outdatedHash,
+	}
+
+	epochClient, err := r.cfg.Notifier.RegisterBlockEpochNtfn(
+		outdatedBlock,
```

Comment:
I don't think we actually need the notifier here, given it'll just end up querying for the missed blocks, and we can do that already given the `chainIO` instance passed in.

Reply:
We can likely just called the `GetClientMissedBlocks` method, though in this case, all we need are the block hashes, since we rely on the chain view to get the filtered blocks directly.

Reply:
If this returns an error, `epochClient` may be nil and `defer epochClient.Cancel()` will panic

Reply:
The `Chain lnwallet.BlockChainIO` interface doesn't satisfy `ChainConn` for `GetClientMissedBlocks`.  Am I missing something?

Reply:
What I mean is, given all it's doing here is grabbing the blocks from the dated block to now we don't actually need the block epoch instance. 

True it doesn't support the interface as is, but you can obtain a block header from a normal block...

Reply:
Hmm sorry but I'm confused about what to do here -- we want to replace RegisterBlockEpochNtfn with GetClientMissedBlocks?